src/modules/Overlay/info.json

```diff
@@ -13,9 +13,12 @@
             "default": 0
         },
         "offset": {
-            "type": "integer",
+            "type": "float",
             "desc": "offset to the output of the step on which the output of the last step is overlayed",
-            "default": -2
+            "default": -2,
+            "min": -2,
```

Comment:
Why is the min just `-2`?

Reply:
Why does it even have a min?

Reply:
Yes you're right, I'll remove this.

Reply:
If you remove the min, range won't work. Let's just make it a `Number` instead. It was fine before I guess :sweat_smile:.

Reply:
Should we use -1 as min?

Reply:
But then the user won't be able to overlay steps before that.

Reply:
Doesn't matter which value we put here because it's going to be overwritten dynamically according to the number of steps in steps array.

https://github.com/blurry-x-face/image-sequencer/blob/8be5ac22ab31d9d5ca73c745372fa40882b64435/src/modules/Overlay/Ui.js#L11

Reply:
Ooh. I didn't know that. Then make it `-1` by default. Also, make the default field `-1` too.

Reply:
I think default field should be `-2` because we overlay image at `-2` on `-1`.

Reply:
Does the code take care of the case when there are only two steps?

Reply:
Yes it does!
https://github.com/blurry-x-face/image-sequencer/blob/8be5ac22ab31d9d5ca73c745372fa40882b64435/src/modules/Overlay/Module.js#L7

Reply:
Ok great! make it `-2`